src/main/java/com/conveyal/datatools/manager/gtfsplus/GtfsPlusTable.java

```diff
@@ -0,0 +1,87 @@
+package com.conveyal.datatools.manager.gtfsplus;
+
+import com.conveyal.gtfs.loader.DateField;
+import com.conveyal.gtfs.loader.DoubleField;
+import com.conveyal.gtfs.loader.IntegerField;
+import com.conveyal.gtfs.loader.ShortField;
+import com.conveyal.gtfs.loader.StringField;
+import com.conveyal.gtfs.loader.Table;
+
+import static com.conveyal.gtfs.loader.Requirement.OPTIONAL;
+import static com.conveyal.gtfs.loader.Requirement.PROPRIETARY;
+import static com.conveyal.gtfs.loader.Requirement.REQUIRED;
+
+public class GtfsPlusTable {
+    public static final Table REALTIME_ROUTES = new Table("realtime_routes", RealtimeRoute.class, PROPRIETARY,
```

Comment:
Each of these tables should probably have javadocs as well

Reply:
I'm not really sure what I would say for each of these tables. We don't really have javadoc for these entities on the GTFS side of things, so I wonder why we would need it here.